x-pack/plugins/security_solution/public/management/pages/policy/view/artifacts/delete_modal/policy_artifacts_delete_modal.test.tsx

```diff
@@ -26,28 +29,48 @@ describe('Policy details artifacts delete modal', () => {
   let renderResult: ReturnType<AppContextTestRender['render']>;
   let mockedContext: AppContextTestRender;
   let exception: ExceptionListItemSchema;
-  let mockedApi: ReturnType<typeof eventFiltersListQueryHttpMock>;
+  let mockedApi: ReturnType<typeof exceptionsListAllHttpMocks>;
   let onCloseMock: () => jest.Mock;
 
   beforeEach(() => {
     policyId = uuid.v4();
     mockedContext = createAppRootMockRenderer();
     exception = getExceptionListItemSchemaMock();
     onCloseMock = jest.fn();
-    mockedApi = eventFiltersListQueryHttpMock(mockedContext.coreStart.http);
+    mockedApi = exceptionsListAllHttpMocks(mockedContext.coreStart.http);
     render = async () => {
+      const type: CreateExceptionListSchema['type'] = (
+        [
+          'endpoint_events',
+          'detection',
+          'endpoint',
+          'endpoint_trusted_apps',
+          'endpoint_host_isolation_exceptions',
+          'endpoint_blocklists',
+        ] as Array<CreateExceptionListSchema['type']>
+      )[Math.floor(Math.random() * 5)];
```

Comment:
Yeah, I thought about that too. I went with random since this modal should be the same for every artifact and that every artifact's delete action does the same time, then it suffices to test it against only one artifact on a test cycle to save time. Also, here the mocking method just needs an API client to work with, it doesn't really matter which artifact's API client it is. Let me know what you think.

Reply:
Yeah, I'm just wondering if at some point we update one of the artifacts with some specifics and the test start failing on CI after merged because when locally and on the changes PR it was using other artifacts with this random choice.

Reply:
I see. I'll update it so that it tests for each kind of list.

Reply:
22ceb726686af9c01c9ba7995449d838345cefad